git/README.md

```diff
@@ -135,3 +136,33 @@ spec:
     persistentVolumeClaim:
       claimName: workspace-pvc
 ```
+
+## `git-batch-merge`
```

Comment:
One minor suggestion here: it might be a good idea to explain the intended usage of this Task.  What's an example use case where this is useful?

Reply:
Added a description

Reply:
Excellent, thank you!